lib/assets/javascripts/cartodb/table/mapview.js

```diff
@@ -877,6 +840,13 @@ cdb.admin.MapTab = cdb.core.View.extend({
 
   },
 
+  getMapView: function() {
+    if (!this.mapView) {
```

Comment:
Yeah, the MapTab does some initialization that would create the mapView later in its lifecycle, but since we need the mapView immediately for the header we have to jump-start that process.